docs/remote_inference_blueprints/standard_blueprints/sagemaker_asymmetric_embedding_standard_blueprint.md

```diff
@@ -0,0 +1,224 @@
+# AWS SageMaker Asymmetric Embedding Model Standard Blueprint
+
+This blueprint demonstrates how to deploy an asymmetric embedding model (multilingual-e5-small) using AWS SageMaker and integrate it with OpenSearch for semantic search. The asymmetric model uses different prefixes for queries and passages to optimize search performance.
```

Comment:
> This blueprint demonstrates how to deploy an asymmetric embedding model (multilingual-e5-small) using AWS SageMaker

Seems this doc doesn't have how to deploy the model to AWS SageMaker. Can you add it ?